frontend/public/components/monitoring/dashboards/single-stat.tsx

```diff
@@ -36,7 +36,7 @@ const SingleStat: React.FC<Props> = ({
       })
       .catch((err) => {
         if (err.name !== 'AbortError') {
-          setError(err);
+          setError(_.get(err, 'json.error', err.message));
```

Comment:
nit: prefer optional chaining

```suggestion
          setError(err.json?.error || err.message);
```

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining